.github/workflows/build.yaml

```
@@ -784,6 +784,16 @@ jobs:
         message-id: build-images-ready
         message-path: build-comment.md
 
+  e2e-smoke-tests:
+    needs:
+    - define-job-matrix
+    - push-main-manifests
+    if: ${{ !cancelled() && needs.push-main-manifests.result == 'success' }}
+    uses: ./.github/workflows/e2e-smoke-tests.yaml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/e2e-smoke-tests.yaml": open /home/jailuser/git/.github/workflows/e2e-smoke-tests.yaml: no such file or directory

(workflow-call)